backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Fresh_ADR_Umsatzliste_BPartner_Report.sql

```diff
@@ -63,8 +63,10 @@ SELECT
 	 ,COALESCE ((SELECT String_Agg(ai_value, ', ' ORDER BY ai_Value) FROM Report.fresh_Attributes WHERE M_AttributeSetInstance_ID = $5), 'alle') AS attributes
 	 ,to_char($1, 'DD.MM.YYYY') AS Base_Period_Start
 	 ,to_char($2, 'DD.MM.YYYY') AS Base_Period_End
-	 ,um.ad_org_id 
-	 FROM report.fresh_umsatzliste_bpartner_report(
+	 ,um.ad_org_id
+     ,COALESCE((SELECT name FROM AD_Org WHERE $7 = AD_Org_ID AND isActive = 'Y'), 'Alle') AS org
```

Comment:
Why would you write 'Alle' when you did not find the org?
Write something like NOT FOUND or - so that the user knows that is a problem.